Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst

```diff
@@ -184,3 +183,20 @@ should not set this argument since you want TCE to use the global
    $dataHandler->process_datamap();
    $dataHandler->process_cmdmap();
 
+.. _tcemain-error-handling:
+
+Error handling
+============================
+
+The data handler has a property `errorLog` as an `array`. In this property, the data handler collects all errors.
+You can use this to log all errors that occur during the data handler would be used.
```

Comment:
I would change "log all errors" to "log or display errors" or just "handle errors" because using the logging framework is just one of many options.